caas/kubernetes/clientconfig/k8s.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/juju/juju/cloud"
 )
 
-var logger = loggo.GetLogger("juju.caas.clientconfig")
+var logger = loggo.GetLogger("juju.kubernetes.clientconfig")
```

Comment:
juju.caas.kubernetes.clientconfig? to keep with the directory structure

Reply:
done